docs/ossm/ambient/README.md

```diff
@@ -0,0 +1,340 @@
+
+**Prerequisites**
+
+* The OpenShift Service Mesh operator is installed.
+* An Istio CNI resource is created.
+
+**Procedure**
+
+1. Add a label to the namespace containing the Istio control plane, for example, the `istio-system` system namespace.
+
+```bash
+$ oc label namespace istio-system istio-discovery=enabled
+```
+
+2. Add a label to the namespace containing the Ztunnel proxy, for example, the `ztunnel` namespace.
+
+```bash
+$ oc label namespace ztunnel istio-discovery=enabled
+```
+
+4. Modify the `Istio` control plane resource to include a `discoverySelectors` section with the same label.
+
+```bash
+$ oc patch istio default --type=merge -p '{
+  "spec": {
+    "values": {
+      "meshConfig": {
+        "discoverySelectors": [
+          {
+            "matchLabels": {
+              "istio-discovery": "enabled"
+            }
+          }
+        ]
+      }
+    }
+  }
+}'
+```
+
+### 3.6 About the Bookinfo Application
+
+Installing the `bookinfo` example application consists of two main tasks: deploying the application and creating a gateway so the application is accessible outside the cluster.
+
+You can use the `bookinfo` application to explore service mesh features. Using the `bookinfo` application, you can easily confirm that requests from a web browser pass through the mesh and reach the application.
+
+The `bookinfo` application displays information about a book, similar to a single catalog entry of an online book store. The application displays a page that describes the book, lists book details (ISBN, number of pages, and other information), and book reviews.
+
+The `bookinfo` application is exposed through the mesh, and the mesh configuration determines how the microservices comprising the application are used to serve requests. The review information comes from one of three services: `reviews-v1`, `reviews-v2` or `reviews-v3`. If you deploy the `bookinfo` application without defining the `reviews` virtual service, then the mesh uses a round robin rule to route requests to a service.
+
+By deploying the `reviews` virtual service, you can specify a different behavior. For example, you can specify that if a user logs into the `bookinfo` application, then the mesh routes requests to the `reviews-v2` service, and the application displays reviews with black stars. If a user does not log into the `bookinfo` application, then the mesh routes requests to the `reviews-v3` service, and the application displays reviews with red stars.
+
+For more information, see [Bookinfo Application](https://istio.io/latest/docs/examples/bookinfo/) in the upstream Istio documentation.
+
+#### 3.6.1 Deploying the Bookinfo application
+
+**Prerequisites**
+
+* You have deployed a cluster on OpenShift Container Platform 4.15 or later.
```

Comment:
In Line 37, we have specified the pre-requisite as OCP 4.19+ Whereas here we are suggesting 4.15, lets be consistent.

```
**OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
```

Reply:
Updated.